packages/shared/src/utils/helpers/dtrader-v2-helpers.ts

```diff
@@ -27,8 +28,17 @@ export const getPositionsV2TabIndexFromURL = () => {
     return 0;
 };
 
-export const isDTraderV2 = () =>
-    !!JSON.parse(localStorage.getItem('FeatureFlagsStore') ?? '{}')?.data?.dtrader_v2 && window.innerWidth < 600;
+export const isDTraderV2Width = () => window.innerWidth < 600;
```

Comment:
Does `isMobile` from `useDevice`  have same width? (<600)
In case if yes, why do we need isDTraderV2Width?